src/assets/assets.json

```diff
@@ -0,0 +1,82 @@
+{"image": 
+ [{
+
+    "key":"sky",
+    "location":"http://labs.phaser.io/assets/skies/space3.png"
+ },
+ {
+    "key":"logo",
+    "location":"http://labs.phaser.io/assets/sprites/phaser3-logo.png"
+ },
+ {
+    "key":"red",
+    "location":"http://labs.phaser.io/assets/particles/red.png"
+ },
+ {
+    "key":"grid_pointer",
+    "location":"sprites/grid_pointer.png"
+ },
+ {
+    "key":"grid_slot",
+    "location":"sprites/slot_cursed.png"
+ },
+ {
+    "key":"map_tiles",
+    "location":"maps/map_tiles.png"
+ },
+ {
+    "key":"stone-tiles",
+    "location":"maps/stone-tiles.jpg"
+ }
+ ],
+ "spritesheet":
+ [{
+     "key:":"soldier",
+     "location":"spritesheets/soldier.png",
+     "frameWidth":300,
+     "frameHeight":300
+  }],
+ "atlas":
+ [{
+     "key":"fire_monster",
+     "location":"spritesheets/fire_monster.png",
+     "json":"spritesheets/fire_monster.json"
+ },
+ {
+     "key":"ice_monster",
+     "location":"spritesheets/ice_monster.png",
+     "json":"spritesheets/ice_monster.json"
+ },
+ {
+     "key":"spider_monster",
+     "location":"spritesheets/spider_monster.png",
+     "json":"spritesheets/spider_monster.json"
+ },
+ {
+     "key":"golem_monster",
+     "location":"spritesheets/golem_monster.png",
+     "json":"spritesheets/golem_monster.json"
+ }],
+ "tileMaps":
+ [{
+     "id":"grass_area",
```

Comment:
For consistency with the above, it might be better to call these `key` instead of ìd`.